src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,52 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<F, S> {
+    private F firstValue;
+    private S secondValue;
 
+    private Pair(F firstValue, S secondValue) {
+        this.firstValue = firstValue;
+        this.secondValue = secondValue;
+    }
+
+    public F getFirst() {
+        return firstValue;
+    }
+
+    public S getSecond() {
+        return secondValue;
+    }
+
+    static <F,S> Pair of(F first, S second) {
```

Comment:
do we need here default access modifier?

Reply:
made public